packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -356,6 +356,9 @@ export class FluidDataStoreRuntime
 	public async request(request: IRequest): Promise<IResponse> {
 		try {
 			const parser = RequestParser.create(request);
+			if (parser.pathParts.length === 0 && request.headers?.viaHandle === true) {
```

Comment:
Maybe use this logic - https://github.com/microsoft/FluidFramework/blob/55bed2d6c67d9344adbc028a4f5703c56c28d219/packages/framework/aqueduct/src/data-objects/pureDataObject.ts#L114.

It handles requests with just query param as well.

Reply:
i'm intentionally not catching as many things here, as i only want this for handles